src/main/java/users/model/User.java

```diff
@@ -1,9 +1,21 @@
 package users.model;
 
+import item.model.Item;
+import java.util.Map;
+
 public class User {
   private String userName;
   private String password;
   private boolean isActive;
+  private Map<Integer,Item> items;
+
+  public Map<Integer, Item> getItems() {
+    return items;
```

Comment:
Returning items like this could violate encapsulation, why? and what is the alternative?